docs/platforms/flutter/troubleshooting.mdx

```diff
@@ -60,3 +60,36 @@ There is an [issue with the Sentry Flutter SDK](https://github.com/getsentry/sen
 This is an [issue](https://github.com/flutter/flutter/issues/135245) with Flutter itself and should be fixed in [Flutter 3.15](https://github.com/flutter/flutter/labels/found%20in%20release%3A%203.15).
 
 For prior versions, you can work around this by configuring the SDK only to take screenshots when the app is in the `resumed` state. To do this, set `SentryFlutterOptions.attachScreenshotOnlyWhenResumed` to `true`.
+
+## Using Flutter multi-view for Web
+
+Multi-view embedding was introduced in Flutter 3.24. In the [Flutter docs](https://docs.flutter.dev/platform-integration/web/embedding-flutter-web) you will find a detailed guide about the usage of this feature.
+
+Using Sentry in a multi-view application is possible, but there are some limitations.
+The following features do not currently support multi-view:
+
+- Screenshots via the `SentryScreenshotWidget` (which is part of the `SentryWidget`)
+- User interaction integration via the `SentryUserInteractionWidget` (which is part of the `SentryWidget`)
+- Window & Device events via the `WidgetsBindingIntegration`
+
+To prevent the `WidgetsBindingIntegration` from loading by default, you will need to remove the integration.
+
+```dart
+// ignore: implementation_imports
+import 'package:sentry_flutter/src/integrations/widgets_binding_integration.dart';
+...
+final integration = options.integrations.firstWhere(
+    (element) => element is WidgetsBindingIntegration);
+options.removeIntegration(integration);
+```
+
+### Enable multi-view in the example app
+
+To make the [sentry flutter example app](https://github.com/getsentry/sentry-dart/tree/main/flutter/example) `multi-view` ready, you will need to modify the `main.dart` and create the `multi_view_app.dart` file next to the `main.dart` and insert the content from the [docs](https://docs.flutter.dev/platform-integration/web/embedding-flutter-web#handling-view-changes-from-dart).
+Modify the `index.html`in the `web` directory and add the `flutter_bootstrap.js` file.
+
+Also note the `NavigatorKey` in the `MaterialApp` widget. This key is a single `GlobalKey` in the `example app` and therefore doesn't support multiple instances. If you rely on this `NavigatorKey` you will need to handle multiple `Keys` and pass them to the appropriate view/instance. For simplicity, we will not use the `NavigatorKey` in the following example.
```

Comment:
I've been reading this a couple times and I think we shouldn't use our example app as a demonstration for several reasons

1. It has a lot of code, `main.dart` alone has 1.2k LoC which would make it even more confusing for a user.
2. It's a sandbox for us devs, while users can of course test it out we generally don't endorse someone to use our example app in the official sentry-docs, it also doesn't necessarily include any best practices, we mostly use it to validate features
3. Users would need to clone it, build and compile sentry flutter, dart and all the other packages before they can start looking into the demonstration code you've posted which are too many steps

So I would suggest instead to use a very minimal code setup (mainly `main.dart`) that anyone can copy and paste